### PR TITLE
Sort list of services from swarm stacks using natural sorting

### PR DESCRIPTION
**Pull request by pszczekutowicz**
**- What I did**
I added sorting to services list returned by command `docker stack services` from Swarm.

**- How I did it**
I did it by copying the code for `docker service ls` command.

**- How to verify it**
Execute command `docker stack services <STACK_NAME>` and list of services should be sorted.

**- Description for the changelog**
Sort list of services from swarm stacks using natural sorting

### CODE REVIEW

File: cli/command/stack/swarm/services.go [hunk 1]
```diff
@@ -3,6 +3,9 @@ package swarm
 import (
 	"context"
 	"fmt"
+	"sort"
+
+	"vbom.ml/util/sortorder"
```
Comment (thaJeztah) [inline cli/command/stack/swarm/services.go]: nit: we don't group imports in three groups (only two), so this should be grouped (and sorted) with the ones below.

**Comment by thaJeztah** [comment]
ping @silvin-lubecki PTAL

File: cli/command/stack/swarm/services.go (hunk 1 above)
Comment (pszczekutowicz) [reply cli/command/stack/swarm/services.go]: Changed.

**Comment by thaJeztah** [comment]
oh! thanks for updating; could you squash those two commits?

**Comment by pszczekutowicz** [comment]
ok, all my commits are squashed.

**Pull request merged**